apps/trading-e2e/src/integration/market-liquidity.cy.ts

```diff
@@ -84,7 +84,7 @@ describe('liquidity table - trading', { tags: '@smoke' }, () => {
     cy.get(rowSelector)
       .first()
       .find(colAverageEntryValuation)
-      .should('have.text', '685,852.93692');
+      .should('have.text', '685,852.94');
```

Comment:
chore (blocking): Can we add a new AC for this https://github.com/vegaprotocol/specs/blob/master/user-interface/5002-LIQP-liquidity.md

Reply:
https://github.com/vegaprotocol/specs/pull/1769